src/Microsoft.Extensions.ProjectModel.Sources/IProjectContext.cs

```diff
@@ -18,6 +18,7 @@ public interface IProjectContext
         string Config { get; }
         string DepsJson { get; }
         string RuntimeConfigJson { get; }
+        string PackageLockFile { get; }
```

Comment:
>  I'm not even that convinced it's worth it. 

Me neither. It's really just mean to find the file, not necessarily to read/parse it.